controllers/auth.js

```diff
@@ -69,9 +73,32 @@ const logout = async (req, res) => {
   });
 };
 
+const avatarsPath = path.join(__dirname, "..", "public", "avatars");
+
+const updateAvatar = async (req, res) => {
+  const { _id } = req.user;
+  const { path: tempPath, originalname } = req.file;
+  console.log(tempPath);
```

Comment:
```suggestion
  console.log(tempPath);
``` Видаліть зайві console.log